version.tf

```diff
@@ -2,16 +2,4 @@ terraform {
   required_version = ">= 1.0.0, <1.7.0"
   # If your module requires any terraform providers, uncomment the "required_providers" section below and add all required providers.
   # Each required provider's version should be a flexible range to future proof the module's usage with upcoming minor and patch versions.
-
-  required_providers {
-    ibm = {
-      source  = "IBM-Cloud/ibm"
-      version = ">= 1.53.0, < 2.0.0"
-    }
```

Comment:
Is IBM provider not required by the Account settings module?

Reply:
pre-commit was flagging an unused provider, but it's not clear why it was saying it's unused so I removed to test it. I can re-add and add an ignore on the unused provider as provider will be needed in the other PR for access groups

Reply:
re-added and added an ignore for tflint